modules/cas_cache/utils/utils_rpool.c

```diff
@@ -120,7 +120,10 @@ void cas_rpool_destroy(struct cas_reserve_pool *rpool_master,
 		return;
 	}
 
-	for (i = 0; i < cpu_no; i++) {
+	for (i = 0; i < max_cpus; i++) {
```

Comment:
If some CPU that was online on CAS load will go offline during CAS operation, here we will have memory leak.
If some CPU that was offline on CAS load will go online during CAS operation, here we will be attempting to deallocate memory we didn't allocated.

Reply:
> If some CPU that was online on CAS load will go offline during CAS operation, here we will have memory leak.
I think this case will be fixed if I remove the two lines below. 

> If some CPU that was offline on CAS load will go online during CAS operation, here we will be attempting to deallocate memory we didn't allocated.
I think this is safe because we allocate memory for num_possible_cpus(0 in cas_rpool_create().

Reply:
Also please advise if my code flow assumptions are wrong since I'm not deeply familiar with the project overall.